test/string-class-test.js

```diff
@@ -0,0 +1,175 @@
+var expect = require('chai').expect;
+require('../public/src/string-class');
+
+describe('String Class', () => {
+
+  describe('hasVowels', () => {
+    it('should return true for strings that has vowels in them', () => {
+      expect('word'.hasVowels()).to.be.true;
+      expect('andela'.hasVowels()).to.be.true;
```

Comment:
### Nit-Pick
Since you are testing the same thing for both line 8 and 9, you may want to remove one out of the two

Also in line 15 and 16, I think there is no need to repeat the same test twice, maybe you can remove one of it also